provider/pkg/provider/provider.go

```diff
@@ -120,6 +123,8 @@ func (k *pulumiserviceProvider) Configure(_ context.Context, req *pulumirpc.Conf
 	}
 	client, err := pulumiapi.NewClient(&httpClient, *token, *url)
 
+	escClient := esc_client.New(fmt.Sprintf("esc-sdk/1 (%s; %s)", version.Version, runtime.GOOS), *url, *token, false)
```

Comment:
Probably want to use a distinct user agent here so that we can distinguish provider traffic from other traffic

```suggestion
	escClient := esc_client.New(fmt.Sprintf("esc-sdk/1 (%s; %s)", version.Version, runtime.GOOS), *url, *token, false)
```

Reply:
Replaced with `"pulumicloud-provider/1 (%s; %s)"`

Reply:
@IaroslavTitov let's keep it the actual name `provider-pulumiservice` or just `provider`.

Reply:
Sounds good, updated